src/ConfigChanges/ConfigFile.js

```diff
@@ -173,14 +200,26 @@ function resolveConfigFilePath (project_dir, platform, file) {
             absolute: true
         }).map(path.normalize);
 
-        if (matches.length) filepath = matches[0];
+        if (matches.length) {
+            filepath = matches[0];
+        }
 
-        // [CB-5989] multiple Info.plist files may exist. default to $PROJECT_NAME-Info.plist
+        // [CB-5989] multiple Info.plist files may exist. default to Info.plist then $PROJECT_NAME-Info.plist
         if (matches.length > 1 && file.includes('-Info.plist')) {
```

Comment:
There might be an issue here because we're checking if `matches.length > 1` and if no `AppName-Info.plist` file exists then we might not have any matches, so maybe this needs to change to `matches.length != 1`

Reply:
Should be fixed by [49e3f4c](https://github.com/apache/cordova-common/pull/212/commits/49e3f4c9a7aa2e041dca6afa84a5d22dc32df301)